### DISCUSSION TITLE
WSA and UDP issues

### DISCUSSION THREAD

**Discussion by johnkellyjr** [microsoft/WSA#64] [General]

**Comment by Mids9** [comment]
https://github.com/microsoft/WSA/discussions/64#discussion-4491085

**Reply by bubbleguuum** [reply]
Did you enable "Advanced networking" in WSA settings ?

**Reply by johnkellyjr** [reply]
Yes, that was one of the first things done, as well as trying it disabled, and all firewalls are disabled.

**Reply by matholliday** [reply]
I'm having similar issues discovering IoT devices on the local network. I can discover a virtual device running on the same machine as WSA (when the FW is off) but not other physical devices on the local network. Presumably WSA is either blocking the UDP broadcast request or the responses from the physical devices. 

Any idea how I enable "Advanced networking" in WSA settings - I'm running 2211.40000.11.0 but there is no option under the System settings?

**Reply by matholliday** [reply]
> Requires Windows11 22H2 or newer

Great, that's done the trick, ta.
